### PR TITLE
Updating parsed plugin list from plugins.yml

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: assets/catalog/parsed_plugin_list.js [hunk 1]
```diff
@@ -139,6 +145,12 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-collector-haproxy"
   },
   {
+    "name": "Sessioninfo",
+    "type": "Collector",
+    "description": "collect Paloalto firewall session info",
```
Comment (kjlyon) [inline assets/catalog/parsed_plugin_list.js]: *Collects, capitalize and add an s.

File: assets/catalog/parsed_plugin_list.js [hunk 2]
```diff
@@ -187,6 +199,12 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-collector-mesos"
   },
   {
+    "name": "MongoDB",
+    "type": "Collector",
+    "description": "Collect statistics and metrics information about MongoDB",
```
Comment (kjlyon) [inline assets/catalog/parsed_plugin_list.js]: *Collects, add s.

File: assets/catalog/parsed_plugin_list.js [hunk 3]
```diff
@@ -313,6 +343,18 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-processor-tag"
   },
   {
+    "name": "Cassandra",
+    "type": "Publisher",
+    "description": "No description available.",
```
Comment (kjlyon) [inline assets/catalog/parsed_plugin_list.js]: Could we say something in the description like, ` "A publisher plugin for Cassandra cluster statistics"`

File: assets/catalog/parsed_plugin_list.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ myfcn(
   {
     "name": "Ping",
     "type": "Collector",
-    "description": "Collects Ping latency metrics",
+    "description": "No description available.",
```
Comment (kindermoumoute) [inline assets/catalog/parsed_plugin_list.js]: Is it possible to keep former description when there is already one in this file?

**Comment by kindermoumoute** [comment]
It makes sense. We can keep it like that and notify those plugin maintainers they must update their plugin description.

**Comment by nanliu** [comment]
I opened an issue for both project, and I'll recreate this PR with the updated descriptions:
https://github.com/IrekRomaniuk/snap-plugin-collector-sessioninfo/issues/5
https://github.com/raintank/snap-plugin-collector-ping/issues/4

**Pull request closed without merging**